src/renderers/testing/ReactTestRenderer.js

```diff
@@ -141,7 +139,6 @@ var ReactTestRenderer = {
 
   /* eslint-disable camelcase */
   unstable_batchedUpdates: ReactUpdates.batchedUpdates,
-  unstable_renderSubtreeIntoContainer: renderSubtreeIntoContainer,
```

Comment:
This shouldn't be in the test renderer according to @spicyj because this currently uses the ReactDOM dependency.